cmd/kyverno/main.go

```diff
@@ -87,7 +90,8 @@ func main() {
 	}
 
 	klog.InitFlags(nil)
-	log.SetLogger(klogr.New())
+	log.SetLogger(klogr.NewWithOptions(klogr.WithFormat(klogr.FormatKlog)))
+	flag.StringVar(&logFormat, "logging-format", "text", "This determines the output format of the logger.")
```

Comment:
Replace `"logging-format"` with `"loggingFormat"` for consistency with other args.

Reply:
Please fix this.